### PR TITLE
Changes for Mura to Masa Scope for 7.3 version of Masa CMS.

### PR DESCRIPTION
**Pull request by jeroenweareorangeeu**

### CODE REVIEW

File: 05_customizing-extending-masa/05_m-tag/01_masa-scope-in-dev.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-permalink: /customizing-extending-masa/m-tag/mura-scope-in-dev/
+permalink: /customizing-extending-masa/m-tag/masa-scope-in-dev/
 ---
 
 # Scope in dev
 
-When developing Masa CMS Themes or Plugins the `m` scope is available. This is referred to as the 'Mura Scope'.
-This scope exists next to the familiair `$` scope ('dollar scope').
+When developing Masa CMS Themes or Plugins the `m` scope is available. This is referred to as the 'Masa Scope'.
+The `$` scope ('dollar scope') and `mura` scope are deprecated and should not be used anymore.
```
Comment (guustnieuwenhuis) [inline 05_customizing-extending-masa/05_m-tag/01_masa-scope-in-dev.md:14]: Could you put this in an alert (warning)?
`{: .alert .alert-warning }`

File: 06_deprecations/02_deprecation_warnings.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Configuration
+There are two settings that control deprecation warnings in Masa CMS:
+
+```markdown
+deprecationwarningsenabled=true
+deprecationlogfile=deprecations
+```
+
+By default `deprecationwarningsenabled` is `true`, enabling logging the deprecation warnings.
```
Comment (guustnieuwenhuis) [inline 06_deprecations/02_deprecation_warnings.md:32]: Can you add that we advice to disable deprecation warnings in production environment? put this message in an alert box.
`{: .alert .alert-warning }`

File: 06_deprecations/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
+layout: default
+title: Deprecations
+nav_order: 7
+has_children: true
+last_modified_date: 13-12-2021 13:00
+permalink: /deprecations/
+---
+
+# Deprecations
```
Comment (guustnieuwenhuis) [inline 06_deprecations/index.md:10]: Could you add a definition of deprecation or an explanation of what it means?

**Review by guustnieuwenhuis — CHANGES_REQUESTED** [review]

File: 06_deprecations/03_mura_deprecations.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+### Deprecated arguments in function ``getAdvancedSearch`` in userGateway.cfc and userManager.cfc
+In the files ``core/mura/user/userGateway.cfc`` and ``core/mura/userGateway.cfc  `` the following arguments are deprecated for the function ``getAdvancedSearch``:
+
+* siteid
+* ispublic
+
+Please remove these arguments in your calls to these functions.
+
+# JS
```
Comment (guustnieuwenhuis) [inline 06_deprecations/03_mura_deprecations.md:132]: this should be a H2 (##)

File: 06_deprecations/03_mura_deprecations.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+## Function ``addLoadEvent`` in mura.js
```
Comment (guustnieuwenhuis) [inline 06_deprecations/03_mura_deprecations.md:133]: this should be a H3 (###)

File: 05_customizing-extending-masa/05_m-tag/01_masa-scope-in-dev.md (hunk 1 above)
Comment (jeroenweareorangeeu) [reply 05_customizing-extending-masa/05_m-tag/01_masa-scope-in-dev.md:16]: Done

File: 06_deprecations/index.md (hunk 3 above)
Comment (jeroenweareorangeeu) [reply 06_deprecations/index.md:10]: Done

File: 06_deprecations/03_mura_deprecations.md (hunk 5 above)
Comment (jeroenweareorangeeu) [reply 06_deprecations/03_mura_deprecations.md]: Done

File: 06_deprecations/03_mura_deprecations.md (hunk 4 above)
Comment (jeroenweareorangeeu) [reply 06_deprecations/03_mura_deprecations.md]: Done

File: 06_deprecations/02_deprecation_warnings.md (hunk 2 above)
Comment (jeroenweareorangeeu) [reply 06_deprecations/02_deprecation_warnings.md:32]: Done

**Review by guustnieuwenhuis — APPROVED** [review]

**Pull request merged**